### DISCUSSION TITLE
Custom Keys & Scoping

### DISCUSSION THREAD

**Discussion by yuiidev** [spatie/laravel-route-attributes#87] [Q&A] [answered]
Does this package support this feature? https://laravel.com/docs/9.x/routing#implicit-model-binding-scoping

Edit: Should have specified that I meant the `Route::get(...)->scopeBindings();` call.

**Comment by freekmurze** [comment, marked as answer]
That is currently not supported, but feel free to create a PR that adds this functionality. Make sure to update the readme and tests.
